qdp/DEVELOPMENT.md

```diff
@@ -105,6 +145,12 @@ python benchmark_dataloader_throughput.py
 python benchmark_e2e.py
 ```
 
+Or use the make command from the `qdp/` directory:
+
+```bash
+make e2e_test
```

Comment:
Our benchmarks are not stable right now, we might use different benchmark command. so I don't think we need this in make.
eg. tryout different qubits`python e2e.py --qubits 100`